docs: create and verify architecture file with CLI (#70)
tier4/CARET_doc: docs/tips/create_and_verify_architecture.md

# Creation and verification of architecture files with CLI

CARET provides CLI to create architecture files and verify paths.

## Creation of architecture file

> The following commands can be used to create an architecture file that has minimum descriptions from trace data.

Reviewer: Architectureはminimum descriptionと言うよりは、静的な（実行毎に依らない）情報を持ったものです。
以下の表現ではいかがでしょうか？
```suggestion
The following commands can be used to create an architecture file that has application structure and latency definition.
```

```bash
ros2 caret create_architecture_file [PATH_TO_CTF] -o [OUTPUT_PATH]
```

After creating the architecture file, please refer to [How to define a target path](https://tier4.github.io/CARET_doc/latest/tutorials/configuration/#how-to-define-a-target-path).

## Verification of paths

> The following command executes the `path.verify()` method on paths defined in the architecture file.

Reviewer: この文章を呼んだ後に、「path.verifyとはなに？」となってしまうので、
「path.verifyが行う内容」を書いたほうが良いと思います。
```suggestion
The following command verifies whether latency definitions are set sufficiently.
```

```bash
ros2 caret verify_paths [PATH_TO_ARCHITECTURE_FILE] -p [DEFINED_PATH_NAMES]
```

- `-p` is optional; by default, `path.verify()` is executed for all defined paths.
- Any number of [DEFINED_PATH_NAMES] can be inputted.

> Please refer to [How to define latency of a single node](https://tier4.github.io/CARET_doc/latest/tutorials/configuration/#how-to-define-latency-of-a-single-node) for details on the `path.verify()` method and how to handle output warnings.

Reviewer: 「このコマンドはverify() を実行しているのと同じです。verifyコマンドについてはこちらをン参照ください」というように、ここで初めてverify()を出すようにしてはいかがでしょうか？

# Creation and verification of architecture files with CLI

CARET provides CLI to create architecture files and verify paths.

## Creation of architecture file

The architecture file describes the application structure and latency definitions.

> The following commands can be used to create a template architecture file including only the application structure section.

Reviewer: こちら細かいですが、architecture file templateにお願いします。

```suggestion
The following commands can create an architecture file template including only the application structure section.
```
